git-checkout-interactive

```diff
@@ -33,5 +33,10 @@ printf "\n${GREEN}Enter the branch number to be checked out: "
 read userInput
 userInput=$((userInput-1))
 
+if [ userInput = "userInput" ]; then
```

Comment:
Can you please explain what line 36 does? How does it verify if the index is invalid?

Reply:
If no value is choosen or an invalid value  like 6 with only 3 branches. 
userInput value is "userInput".

So I compare string, maybe there is a better solution. This is the first I found.

Reply:
It's not working. It gives an error for valid inputs also